contracts/src/v0.8/automation/dev/MercuryRegistry.sol

```diff
@@ -144,6 +144,14 @@ contract MercuryRegistry is ConfirmedOwner, AutomationCompatibleInterface, Strea
     return (filteredValues.length > 0, performData);
   }
 
+  function checkErrorHandler(
+    uint256 errCode,
+    bytes memory extraData
+  ) external view override returns (bool upkeepNeeded, bytes memory performData) {
+    // dummy function, just return true and extraData
+    return (true, extraData);
+  }
+
```

Comment:
default implementation should be false to keep existing behaviour

return (false, new bytes(0))

Also you might be able to define this within the interface itself without requiring all contract code to change

Reply:
well, just checked : Functions in interfaces cannot have an implementation.